soc/nordic/nrf54l/soc.c

```diff
@@ -149,7 +151,12 @@ static int nordicsemi_nrf54l_init(void)
 		nrf_power_task_trigger(NRF_POWER, NRF_POWER_TASK_CONSTLAT);
 	}
 
+#if defined(CONFIG_SOC_NRF54L_VREG_MAIN_DCDC)
 	if (IS_ENABLED(CONFIG_SOC_NRF54L_VREG_MAIN_DCDC)) {
```

Comment:
What is the reason to keep this branch if the configuration options was removed?

Reply:
I reverted this change and remove kconfig option in another PR

Reply:
see: https://github.com/zephyrproject-rtos/zephyr/pull/77160